src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -250,6 +252,10 @@
                    (if success?
                      (swap! state assoc :locked? (get-in (get-parsed-response) ["workspace" "isLocked"]))
                      (swap! state assoc :error status-text)))})
+     (endpoints/call-ajax-orch
+       {:endpoint (endpoints/get-refresh-token-date)
+        :on-done (fn [{:keys [success?]}]
+                   (swap! state assoc :has-refresh-token? success?))})
```

Comment:
To @davidangb's point, I think this is exactly the right check here, since the real check is done in the API on login, so verifying the date would be duplicated code. However, since I'm naturally inclined to ask the question, "Why is this only checking for success?" a comment like:
```
;; Only check for the existence of this key, since the validity is checked by the API on login
```
would be helpful.